exonum/src/node/consensus.rs

```diff
@@ -389,6 +391,16 @@ impl NodeHandler {
         // Lock to propose
         if self.state.locked_round() < prevote_round && self.state.propose(&propose_hash).is_some()
```

Comment:
Micro-nit: I think `if let Some(propose_state) = self.state.propose(&propose_hash)` would be more idiomatic.